safenode/src/client/wallet.rs

```diff
@@ -30,9 +32,15 @@ impl<W: SendWallet> WalletClient<W> {
     }
 
     /// Send tokens to another wallet.
-    pub async fn send(&mut self, amount: Token, to: PublicAddress) -> Result<()> {
-        let _dbcs = self.wallet.send(vec![(amount, to)], &self.client).await?;
-        Ok(())
+    pub async fn send(&mut self, amount: Token, to: PublicAddress) -> Result<Vec<CreatedDbc>> {
+        let dbcs = self.wallet.send(vec![(amount, to)], &self.client).await?;
+        // self.wallet.store().await;
```

Comment:
Any future ideas for this commented line?

Reply:
aah... yeah... need to fix, let me see if I planned it for now or follow up..